README.md

```diff
@@ -223,18 +223,18 @@ jobs:
         if: steps.cache-primes.outputs.cache-hit != 'true'
         run: ./generate-primes -d prime-numbers
 ```
-## Cache segment restore timeout
 
+## Known practices and workarounds
```

Comment:
Would it make sense to create separate file for this?

Reply:
I think it would yes. We can move it to another file.